src/services/StorageService.ts

```diff
@@ -0,0 +1,97 @@
+import AWS from 'aws-sdk';
+
+const s3 = new AWS.S3({ region: process.env.BUCKET_REGION });
+export class StorageService {
+  async uploadFile(
+    fileName: string,
+    file: Express.Multer.File,
+    options?: Object
+  ): Promise<string | null> {
+    let fileNameKey = fileName;
+    if (options) {
+      if ('appendFileName' in options) {
+        fileNameKey = `${fileName}_${options['appendFileName']}`;
+      }
+    }
+    const params = {
+      Bucket: process.env.BUCKET_NAME,
+      Key: fileNameKey,
+      Body: file.buffer,
+      ContentDisposition: `attachment; filename="${fileName}${file.originalname.substring(

```

Comment:
could we add a comment for this? looks like it's trying to get the filename without the extension; any specific reason for that?

Reply:
done